### DISCUSSION TITLE
Python Version

### DISCUSSION THREAD

**Discussion by TWIISTED-STUDIOS** [vladmandic/sdnext#387] [Q&A]
My System has multiple Python versions all installed for versions reasons, they are all registered in the paths so can be used anytime I please. I have my priority python set as 3.11. 

So my question is can the installer detect the different versions of python installed and use the correct one or would I have to change the priority order list?

**Comment by vladmandic** [comment]
There are tools that do that. I don't use them, so not going to name specific one.

**Reply by 1r00t** [reply]
For me it took the wrong python version while installing. I needed to specify 3.10 in the webui.sh as `python_cmd`.

**Reply by SHARKASUR** [reply]
thing were going fine but at the final installation movement after it asks for v1.5 model download in the final step it shows python is not found and shows error 9009 . i hsve python 3.10 which was working fine on normal SD . i have no idea about coding and stuff so can anyone please help me ...please explain it in simplesteps if you can
